### PR TITLE
feat(self): no_collision

### PR DESCRIPTION
**Pull request by Aure7138**

### CODE REVIEW

File: BigBaseV2/src/backend/looped/self/no_collision.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+	void looped::self_no_collision()
+	{
+		if (g_local_player == nullptr) return;
+
+		bool bNoCollsion = g->self.no_collision;
+
+		auto set_collison = [](float value) 
+		{
+			static int offsets[] = { 0x30, 0x10, 0x20, 0x70, 0x00, 0x2C };
```
Comment (xiaoxiao921) [inline BigBaseV2/src/backend/looped/self/no_collision.cpp]: Let's not do that and use a struct instead

File: BigBaseV2/src/backend/looped/self/seat_belt.cpp [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#include "backend/looped/looped.hpp"
+
+namespace big
+{
+	static bool bLastSeatBelt = false;
+
+	void looped::self_seat_belt()
```
Comment (xiaoxiao921) [inline BigBaseV2/src/backend/looped/self/seat_belt.cpp]: I thought we already had seat belt under Vehicle ?

**Review by xiaoxiao921 — CHANGES_REQUESTED** [review]

File: BigBaseV2/src/backend/looped/self/seat_belt.cpp (hunk 2 above)
Comment (Aure7138) [reply BigBaseV2/src/backend/looped/self/seat_belt.cpp]: my bad, didnt check vehicle view

File: BigBaseV2/src/backend/looped/self/no_collision.cpp (hunk 1 above)
Comment (Aure7138) [reply BigBaseV2/src/backend/looped/self/no_collision.cpp]: Ok, will make a PR to GTAV Class in quick

**Comment by Aure7138** [comment]
Yes, I think it's done

**Review by xiaoxiao921 — APPROVED** [review]

**Review by Yimura — APPROVED** [review]

**Pull request merged**